public/js/loginValidation.js

```diff
@@ -0,0 +1,62 @@
+
+function isValidEmail(elementID) {
```

Comment:
This is being removed?

Reply:
no our function is picking up invalid email that html missed (were .com was missing)